packages/wonder-blocks-modal/docs.md

```diff
@@ -15,6 +16,64 @@ When a dialog opens, focus moves to an element inside the dialog. See notes belo
     - If focus is on the first tabbable element inside the dialog, moves focus to the last tabbable element inside the dialog.
 - Escape: Closes the dialog.
 
+#### Initial focus placement:
+The initial focus placement depends on the following scenarios:
+
+1. `initialFocusId` (default): `ModalLauncher` exposes this prop as a string. The dialog will try to find this element into the DOM. If it's found, focus is initially set on this element.
+2. focusable elements: This is the second scenario, where the dialog tries to find the first ocurrence of possible focusable elements.
+3. Dialog: If the first two conditions are not met, then focus is initially set to the Dialog element itself.
+
+### Example: Set initial focus on a given element inside the modal
+
+```js
+const React = require("react");
+
+const {Title} = require("@khanacademy/wonder-blocks-typography");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const {Strut} = require("@khanacademy/wonder-blocks-layout");
+const Spacing = require("@khanacademy/wonder-blocks-spacing").default;
+
+const modalInitialFocus = ({closeModal}) => (
+    <OnePaneDialog
+        title="Single-line title"
+        content={
+            <View>
+                <View>
+                    <label>Label</label>
+                    <input type="text" />
+                    <Strut size={Spacing.medium} />
+                    <Button data-initial-focus>
+                        Button to receive initial focus
+                    </Button>
+                </View>
+            </View>
+        }
+        footer={
+            <React.Fragment>
+                <Button kind="tertiary" onClick={closeModal}>
+                    Cancel
+                </Button>
+                <Strut size={Spacing.medium} />
+                <Button>
+                    Submit
+                </Button>
+            </React.Fragment>
+        }
+    />
+);
+
+<View style={{flexDirection: "row"}}>
+    <ModalLauncher
+        onClose={() => window.alert("you closed the modal")}
+        initialFocusId="[data-initial-focus]"
```

Comment:
This is a little confusing.  I was expecting to specify an id not a selector.

Reply:
That's a valid point. Using IDs makes total sense b/c that way we should only be looking for one single node.

On the other side, we could use selectors, and add some documentation to recommend devs to use custom data attributes tight to WB (?). 

What do you think it would be the best approach?

1. initialFocusId="some-specific-id", then inside ModalBackdrop using `node.querySelector(`#${initialFocusId}`)` b/c for some weird reason `getElementId` is not included in `HTMLElement`.
vs
2. initialFocusId="[data-wb-modal-initial-focus]" and adding some extra documentation to recommend devs following these kind of naming conventions. For reference you can see what cypress does (https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements).

Reply:
My vote is for the option 1.

Reply:
Here it is :) 
https://github.com/Khan/wonder-blocks/pull/402/files#diff-0a507f47a429306bd08c332dda512cb6R70